Tchap/Modules/Room/RoomViewController.m

```diff
@@ -215,6 +212,9 @@ @interface RoomViewController () <UIGestureRecognizerDelegate, Stylable, RoomTit
 // Direct chat target user when create a discussion without associated room
 @property (nonatomic, strong) User *discussionTargetUser;
 
+// Observe kRiotDesignValuesDidChangeThemeNotification to handle user interface theme change.
+@property (nonatomic, weak) id kRiotDesignValuesDidChangeThemeNotificationObserver;
```

Comment:
It's the opportunity to rename this property and remove the "k" prefix (used for constants). It could be `themeDidChangeNotificationObserver`.